### PR TITLE
feat(sf): remove bootstrap, clean up of variables and less files

### PR DESCRIPTION
**Pull request by katjamueller**
* removed bootstrap variables and classes
* removed obsolete classes 
* renamed 'alto-theme' variables and classes
* first steps for responsive dali grid
* new folder structure for stylesheets and dust templates

### CODE REVIEW

**Comment by katjamueller** [comment]
review done by Verana and me, @depoulo could you please QA our changes

File: stylesheets/components/breadcrumb.less [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  list-style: none;
+  font-size: 13px;
+  font-weight: 300;
+  color: lighten(@brand-text, 60%);
+
+  > li {
+    display: inline-block;
+
+    + li:before {
+      content: "/"; // Unicode space added since inline-block means non-collapsing white-space
```
Comment (depoulo) [inline stylesheets/components/breadcrumb.less]: is the space invisible?

File: stylesheets/components/mixins.less [hunk 2]
```diff
@@ -0,0 +1,25 @@
+//
+// Mixin for hover state on buttons
+.button-hover-color(@background-color: darken(@button-background-color-default, 10%), @text-color: @button-text-color-default) {
+  &:hover:not(:disabled):not(.ep-button-disabled) {
```
Comment (depoulo) [inline stylesheets/components/mixins.less]: Maybe consider using [named parameters](http://lesscss.org/features/#mixins-parametric-feature-named-parameters).
Also consider: [_"Parameters are either semicolon or comma separated. It is recommended to use semicolon."_](http://lesscss.org/features/#mixins-parametric-feature-mixins-with-multiple-parameters)

File: stylesheets/components/forms.less [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+
+
+// ---------- Styling ---------- //
+
+//
+// Placeholder mixin
+.input-placeholder-color() {
+  ::-webkit-input-placeholder {color: @input-placeholder-color;}
+  ::-moz-placeholder {color: @input-placeholder-color;}
+  :-ms-input-placeholder {color: @input-placeholder-color;}
```
Comment (depoulo) [inline stylesheets/components/forms.less]: this is handled by [Autoprefixer](https://github.com/18F/C2/pull/657). Just use the standard way [like here](https://github.com/ePages-de/epages-ui/blob/develop/source/brix/brix.core.render/styles/dali/plugin-text.less#L199).

File: stylesheets/components/buttons.less [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+  background-color: @button-background-color-default;
+  color: white;
+  .button-hover-color(darken(@button-background-color-default, 20%), @button-text-color-default); // Hover
+}
+
+// Primary
+.button-primary {
+  background-color: @button-background-color-primary;
+  color: @button-text-color-primary;
+  .button-hover-color(darken(@button-background-color-primary, 10%), @button-text-color-primary); // Hover
```
Comment (depoulo) [inline stylesheets/components/buttons.less]: Actually I meant these: You can just write `.button-hover-color(@background-color: darken(@button-background-color-default, 20%));`. By using the named parameters you don't need to comment what is what.

File: stylesheets/components/grid.less [hunk 5]
```diff
@@ -0,0 +1,12 @@
+.container-fluid {
+  .container-fixed();
+}
```
Comment (depoulo) [inline stylesheets/components/grid.less]:
```
:¨·.·¨:
`·. ŁθŁ
```

File: stylesheets/components/normalize.less [hunk 6]
```diff
@@ -0,0 +1,428 @@
+/*! normalize.css v3.0.2 | MIT License | git.io/normalize */
```
Comment (depoulo) [inline stylesheets/components/normalize.less]: why isn't this in `vendor`?

File: stylesheets/layout.less [hunk 7]
```diff
@@ -353,5 +351,6 @@
   padding-left: 0;
   font-size: 13px;
   font-weight: 300;
-  color: lighten(@alto-theme-brand-text, 60%);
+  color: lighten(@brand-text, 60%);
 }
+*/
```
Comment (depoulo) [inline stylesheets/layout.less]: guess this can be deleted?

File: stylesheets/components/mixins.less [hunk 8]
```diff
@@ -0,0 +1,25 @@
...
+  }
+  &:after {
+    clear: both;
+  }
+}
+
+// Box sizing
+.box-sizing(@boxmodel) {
+  box-sizing: @boxmodel;
+}
```
Comment (depoulo) [inline stylesheets/components/mixins.less]: :D

File: stylesheets/components/dali.less [hunk 9]
```diff
@@ -0,0 +1,68 @@
...
+.dali-grid:not(.dali-grid-editview) .dali-grid-row {
+  flex-wrap: wrap;
+}
+
+// Column
+.dali-grid:not(.dali-grid-editview) .dali-grid-column {
+  flex: 1 0 0;
+  @media (max-width: @screen-md) {
+    display: block;
+    flex: 1 0 47%;
```
Comment (depoulo) [inline stylesheets/components/dali.less]: why exactly 47?

**Comment by depoulo** [comment]
review & QA is fine, modulo my annotations, but there's nothing really important. Just do what you want with'em and then just squash and merge.

**Pull request merged**